tests/js/JSTest.php

```diff
@@ -601,8 +605,7 @@ function foo(a,b){return a/b}',
         $tests[] = array(
             'var timeRegex = /^([2][0-3]|[01]?[0-9])(:[0-5][0-9])?$/
 if (start_time.match(timeRegex) == null) {}',
-            'var timeRegex=/^([2][0-3]|[01]?[0-9])(:[0-5][0-9])?$/
-if(start_time.match(timeRegex)==null){}',
+            'var timeRegex=/^([2][0-3]|[01]?[0-9])(:[0-5][0-9])?$/;if(start_time.match(timeRegex)==null){}',
```

Comment:
As mentioned earlier: the way this was achieved was unsafe, it could break other things.
And it doesn't even matter whether it's a `\n` or a `;`, both of them are only 1 char, both of them are the same size :)